configs/ppo_randomwalks.yml

```diff
@@ -0,0 +1,47 @@
+model:
+  model_path: "randomwalks/1M"
+  tokenizer_path: "randomwalks/1M"
```

Comment:
Update the paths to point to CarperAI HuggingFace model path.

Reply:
done